### PR TITLE
Docs audit: Tags

### PR DESCRIPTION
**Pull request by dbrian57**
Updates and clarifies doc strings for Tag commands. Also add examples for each command.

### CODE REVIEW

File: commands/tags.go [hunk 1]
```diff
@@ -31,33 +31,36 @@ func Tags() *Command {
 			Short: "Display commands to manage tags",
 			Long: `The sub-commands of ` + "`" + `doctl compute tag` + "`" + ` manage the tags on your account.
 
-A tag is a label that can be applied to a resource (currently Droplets, images,
-volumes, volume snapshots, and database clusters) in order to better organize or
```
Comment (andrewsomething) [inline commands/tags.go:35]: The fact that only Droplets, images, volumes, volume snapshots, and database clusters are taggable resources seems like useful info that we should retain somewhere. Maybe in the `apply` command's help?

**Review by andrewsomething — COMMENTED** [review]
Looks good, but I did have one question inline.

File: commands/tags.go (hunk 1 above)
Comment (dbrian57) [reply commands/tags.go:35]: You're absolutely right. I'll work that back in there.

**Review by andrewsomething — APPROVED** [review]
:pray: Thanks Dan!

**Pull request merged**